common.go

```diff
@@ -0,0 +1,125 @@
+package redis_timeseries_go
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+//go:generate stringer -type=AggregationType
+type AggregationType string
+
+const (
+	AvgAggregation   AggregationType = "AVG"
+	SumAggregation   AggregationType = "SUM"
+	MinAggregation   AggregationType = "MIN"
+	MaxAggregation   AggregationType = "MAX"
+	CountAggregation AggregationType = "COUNT"
+	FirstAggregation AggregationType = "FIRST"
+	LastAggregation  AggregationType = "LAST"
+	StdPAggregation  AggregationType = "STD.P"
+	StdSAggregation  AggregationType = "STD.S"
+	VarPAggregation  AggregationType = "VAR.P"
+	VarSAggregation  AggregationType = "VAR.S"
+)
+
+var aggToString = []AggregationType{AvgAggregation, SumAggregation, MinAggregation, MaxAggregation, CountAggregation, FirstAggregation, LastAggregation, StdPAggregation, StdSAggregation, VarPAggregation, VarSAggregation}
+
+type CreateOptions struct {
+	Uncompressed   bool
+	RetentionMSecs time.Duration
+	Labels         map[string]string
+}
+
+var DefaultCreateOptions = CreateOptions{
+	Uncompressed:   false,
+	RetentionMSecs: 0,
+	Labels:         map[string]string{},
+}
+
+// Client is an interface to time series redis commands
+type Client struct {
+	Pool ConnPool
+	Name string
+}
+
+//
+//const TimeRangeMinimum = 0
+//const TimeRangeMaximum = math.MaxInt64
+//const TimeRangeFull = int64(-1)
```

Comment:
?

Reply:
For some reason I comment it out. Fixing on new commit :)